File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        ans = 0 
        for i in words :
            x = 0 
            for j in i : 
                if j  in allowed :
                     x += 1
            if x == len(i) :
                ans+=1
        return ans

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int ans = 0;
        for(auto &i : words){
            int x = 0;
            for(auto &j :  i){
                if(find(allowed.begin(),allowed.end(),j) != allowed.end()) x+=1;
            }
            if(x == i.size()) ans+=1;
        }
        return ans;
    }
};